tracing-subscriber/src/layer.rs

```diff
@@ -770,6 +770,44 @@ impl<'a, S: Subscriber> Context<'a, S> {
             .map(|s| s.exists(id))
             .unwrap_or(false)
     }
+
+    /// Visits every span in the current context with a closure.
+    ///
+    /// The provided closure will be called first with the current span,
```

Comment:
This comment is verbatim from `visit_spans` - wanted to double-check that it isn't accurate and the code does the reverse, iterating starting from root and ending with the current span? I can update it if needed.

Reply:
The comment I wrote is _accurate_, but it's somewhat misleading. Since I'm working on docs anyways, I'll update it to be less misleading.